e2e-tests/cypress/e2e/journeys/insurance/your-buyer/working-with-buyer/working-with-buyer.spec.js

```diff
@@ -0,0 +1,118 @@
+import { saveAndBackButton, submitButton } from '../../../../pages/shared';
+import partials from '../../../../partials';
+import { workingWithBuyer } from '../../../../pages/insurance/your-buyer';
+import { BUTTONS, PAGES } from '../../../../../../content-strings';
+import { ROUTES } from '../../../../../../constants';
+import { YOUR_BUYER as FIELD_IDS } from '../../../../../../constants/field-ids/insurance/your-buyer';
+import { INSURANCE_ROOT } from '../../../../../../constants/routes/insurance';
+import { YOUR_BUYER_FIELDS as FIELDS } from '../../../../../../content-strings/fields/insurance/your-buyer';
+
+const CONTENT_STRINGS = PAGES.INSURANCE.YOUR_BUYER.WORKING_WITH_BUYER;
+
+const {
+  WORKING_WITH_BUYER: {
+    CONNECTED_WITH_BUYER,
+    TRADED_WITH_BUYER,
+  },
+} = FIELD_IDS;
+
+const { COMPANY_OR_ORGANISATION, WORKING_WITH_BUYER, CHECK_YOUR_ANSWERS } = ROUTES.INSURANCE.YOUR_BUYER;
+
+const { taskList } = partials.insurancePartials;
+
+const task = taskList.prepareApplication.tasks.buyer;
+
+context('Insurance - Your Buyer - Working with buyer page - As an exporter, I want to confirm my buyer details', () => {
+  let referenceNumber;
+  let url;
+  let checkYourAnswersUrl;
+
+  before(() => {
+    cy.navigateToUrl(ROUTES.INSURANCE.START);
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    task.link().click();
+
+    cy.completeAndSubmitCompanyOrOrganisationForm();
+
+    cy.getReferenceNumber().then((id) => {
+      referenceNumber = id;
+
+      url = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${WORKING_WITH_BUYER}`;
+      checkYourAnswersUrl = `${Cypress.config('baseUrl')}${INSURANCE_ROOT}/${referenceNumber}${CHECK_YOUR_ANSWERS}`;
+
+      cy.url().should('eq', url);
+    });
+  });
+
+  beforeEach(() => {
+    Cypress.Cookies.preserveOnce('_csrf');
+    Cypress.Cookies.preserveOnce('exip-session');
+  });
+
+  it('renders core page elements', () => {
+    cy.corePageChecks({
+      pageTitle: CONTENT_STRINGS.PAGE_TITLE,
+      currentHref: `${INSURANCE_ROOT}/${referenceNumber}${WORKING_WITH_BUYER}`,
+      backLink: `${INSURANCE_ROOT}/${referenceNumber}${COMPANY_OR_ORGANISATION}`,
+      lightHouseThresholds: {
+        'best-practices': 92,
```

Comment:
Maybe don't need to reduce `best-practices` threshold here?

Reply:
will double check with a push, I think it is lower because of the component and conditional html